File: Python/Function/07.factorial.py
#Write a Function to calculate Factorial of any Number

N = int(input("Enter any number: "))

def factorial(num):
    fact = 1
    
    for i in range(1,num+1):
        fact = fact*i
        
    return fact
    
    
result = factorial(N)

print("Factorial of given number is: ", result)

File: CPP/Function/07.factorial.cpp
//Write a Function to calculate Factorial of any Number

#include <iostream>

using namespace std;

int factorial(int num){
    int fact = 1;
    
    for (int i = 1; i<num+1; i++){
        fact = fact*i;
    }
    
    return fact;
}

int main()
{
    
    float a;
    cout<<"Enter the number: ";
    cin>>a;

    cout<<"Factorial of the number is: "<< factorial(a);
    
    return 0;
}